chibuezeayogu/pencil_backend
# Implement questions and topics model

**PR Description**
- The PR implements `questions` and `topics` model

**This change addresses the need by:** 
 - Add connect application to `Mongodb` using `mongoose ORM`
 - Add `Question` and `Topic` model

**How should this be manually tested?**
* git pull and checkout into `implement-questions-and-topics-model`
* Install packeges using `yarn install`
* Using .env.sample to create a .env file and provide the ENV variables provided
* Start application using `yarn run dev-server`
* Ensure application 

**Relevant Trello links (if applicable)**
* N/A

**Deployment Actions NEEDED**
* N/A

**Screenshots**
* N/A
